tools/sdk-generation-pipeline/ci.yml

```diff
@@ -43,16 +43,8 @@ stages:
   - stage: InstallAndBuild
     jobs:
       - job: Build
-        strategy:
-          matrix:
-            linux:
-              imageName: 'ubuntu-latest'
-            mac:
-              imageName: 'macos-latest'
-            windows:
-              imageName: 'windows-latest'
         pool:
-          vmImage: $(imageName)
+          vmImage: 'ubuntu-latest'
```

Comment:
We will want to use our 1ES agent pools.

Reply:
I have changed it to
```
pool:
          name: "azsdk-pool-mms-ubuntu-2004-perf"
          vmImage: "MMSUbuntu20.04"
```
Is it the one you want? thx

Reply:
It should be 'azsdk-pool-mms-ubuntu-2004-general'